legal-api/src/legal_api/resources/business/business_comments.py

```diff
@@ -0,0 +1,114 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Searching on a business entity.
+
+Provides all the search and retrieval from the business entity datastore.
+"""
+import datetime
+from http import HTTPStatus
+from typing import Tuple
+
+from flask import g, jsonify, request
+from flask_restplus import Resource, cors
+
+from legal_api.exceptions import BusinessException
+from legal_api.models import Business, Comment, User, db
+from legal_api.services import authorized
+from legal_api.services.comments import validate
+from legal_api.utils.auth import jwt
+from legal_api.utils.util import cors_preflight
+
+from .api_namespace import API
+# noqa: I003; the multiple route decorators cause an erroneous error in line space counting
+
+
+@cors_preflight('GET, POST')
+@API.route('/<string:identifier>/comments', methods=['GET', 'POST', 'OPTIONS'])
+class BusinessCommentResource(Resource):
+    """Business Comment service."""
+
+    @staticmethod
+    @cors.crossdomain(origin='*')
+    @jwt.requires_auth
+    def get(identifier):
```

Comment:
get by id is something that exists for all other objects. May have to check whether that should be supported

Reply:
id of comments?

Reply:
ya

Reply:
as per the requirement this is not required. but if we follow that standard i can include it

Reply:
I think get all, get by id is more of a std for an endpoint

Reply:
will include that